en/08_Changelogs/6.0.0.md

```diff
@@ -107,6 +109,17 @@ The code in `silverstripe/externallinks`, `silverstripe/security-report`, and `s
 
 CMS 6 compatible versions of `silverstripe/blog`, `silverstripe/subsites`, and `silverstripe/crontask` have been released with CMS 6.0.0, though they are not commercially supported.
 
+### New supported modules
+
+Some new modules have been created and are now covered by our [supported modules policy](/project_governance/supported_modules/). These modules will be supported for the full lifetime of Silverstripe CMS 6.
+
+> [!WARNING]
+> To continue using TinyMCE as an HTML editor in your existing project, you must add `silverstripe/htmleditor-tinymce` as an explicit dependency.
+
+- [`silverstripe/startup-theme`](https://packagist.org/packages/silverstripe/startup-theme). See [new default front-end theme](#theme) below for more details.
+- [`silverstripe/template-engine` module](https://packagist.org/packages/silverstripe/template-engine). See [abstraction of template rendering](#view-layer-abstraction) for more details.
+- [`silverstripe/htmleditor-tinymce`](https://packagist.org/packages/silverstripe/htmleditor-tinymce). See [TinyMCE is in a separate module now](#tinymce) for more details.
```

Comment:
This should be the first point since it's what people need to pay the most attention to

Reply:
Which "this" is "this"? Do you mean the third item in the list should be moved to the top?

Reply:
The `silverstripe/htmleditor-tinymce` should be the first bullet point as it's the one that requires the most action. People will pay more attention to things at the top of a list

Reply:
Done. They're in alpha order now which is also sensible.